### PR TITLE
ConnectAd Bid Adapter: Sync endpoint Update

### PR DESCRIPTION
**Pull request by rtuschkany**
## Type of change
- [X] Bugfix
- [X] Feature
- [ ] Other

## Other information
Includes https://github.com/prebid/Prebid.js/issues/6361 fix

### CODE REVIEW

**Review by patmmccann — APPROVED** [review]

**Review by patmmccann — APPROVED** [review]

**Review by robertrmartinez — APPROVED** [review]
one thing but not required

**Comment by patmmccann** [comment]
@rtushkany it seems you've disappeared. What do you want to do here?

File: modules/connectadBidAdapter.js [hunk 1]
```diff
@@ -72,24 +86,55 @@ export const spec = {
...
       deepSetValue(data, 'user.ext.eids', validBidRequests[0].userIdAsEids);
     }
 
+    const tid = deepAccess(bidderRequest, 'ortb2.source.tid')
+    if (tid) {
+      deepSetValue(data, 'source.tid', tid)
+    }
+
+    // Set tmax value based on prebid config
+    if (bidderRequest?.timeout) {
```
[the remark this answers is not in the document]
Comment (patmmccann) [reply modules/connectadBidAdapter.js]: You're right, I fixed it

**Comment by robertrmartinez** [comment]
hi @rtuschkany can you resolve conflicts or close PR if it has been done in another?

**Pull request merged**